src/app/legacy/containers/Navigation/index.jsx

```diff
@@ -128,6 +129,12 @@ const NavigationContainer = () => {
       dir={dir}
       script={script}
       service={service}
+      {...(!isAmp &&
+      !isLiveEnv() &&
```

Comment:
Would be good if someone can confirm that this will work! If this is the way it has been used before?

Reply:
There should be a dedicated `isTest` utility: `import isTest from '#app/lib/utilities/isTest';`

Reply:
This seems to make it disappear in the local environment so we can't view it on-branch on localhost